p2p/src/lib.rs

```diff
@@ -191,6 +193,7 @@ where
         }
         {
             let consensus_handle = consensus_handle.clone();
+            let mempool_handle = mempool_handle.clone();
```

Comment:
This can be removed. Just keep the parameter for this function unused.

Reply:
Done.